.github/workflows/actions.yml

```diff
@@ -15,10 +15,8 @@ jobs:
         shell: bash -l {0}
     steps:
     - uses: actions/checkout@v3
-    - uses: conda-incubator/setup-miniconda@v2
+    - uses: mamba-org/setup-micromamba@v1
```

Comment:
I'm kinda surprised this didnt require the channels to be listed like the conda-build job.  Otherwise if it works it works.